docs/guide/docs/quickstart.md

```diff
@@ -28,22 +31,24 @@ All you have to do is pick a name for your bot, a unique `Bot ID` will be genera
 
 You may start from scratch with the Empty bot, or you can use one of the available templates for a quick demonstration of various features.
 
+![Create Bot](assets/Create-Bot.png)
+
 Once created, click on the bot's name to open the Studio interface and edit your bot.
 
-## Studio
+## Studio Dashboard
 
 The Studio is the main interface you'll use to build and edit your bot. From here, among other things, you can:
 
 - Train an NLU model
 - Create dialog flows
 - Manage content
 - Test and debug your bot
-
-You will notice on the left there are a couple of sections available. Some of these are built inside Botpress (like _Content_ and _Flows_), where some are there because of modules installed. Indeed, Botpress is a highly modular platform and the majority of the features you will use are provided by the modules themselves. You may use experimental ones, or even develop your own modules.
+![Botpress Studio](assets/studio.png)
+You will notice on the left menu there are a couple of sections available. Some of these are built inside Botpress (like _Content_ and _Flows_), while others are there because of modules installed. Indeed, Botpress is a highly modular platform, and the majority of the features you will use are provided by the modules themselves. You may use experimental ones, or even develop your own modules.
 
 ### Test your bot
 
-There is a chat emulator built in the studio with which you can test your bot. You can open this emulator from the button in the top right corner, or press 'E' while in the studio. It is made to represent what your visitors will face when they speak with your bot. You can reset the convertsation by clicking the reset button, or you can also start a new conversation with your bot with the conversations button.
+There is a chat emulator built in the studio with which you can test your bot. You can open this emulator from the button in the top right corner, or press 'E' while in the studio. It is made to represent what your visitors will face when they speak with your bot. You can reset the conversation by clicking the reset button, or you can also start a new conversation with your bot with the conversations button.
```

Comment:
I think we should provide a complete documentation section on the emulator/debugger as there's a lot going on there. (i.e how to inspect a message, processing, logs, etc.)

Reply:
There is an emulator section. Not yet started real work on that but getting there. Did a few cosmetic changes though.